src/pages/AccountPage.jsx

```diff
@@ -0,0 +1,253 @@
+/* eslint-disable react/no-children-prop */
+/* eslint-disable no-unused-vars */
+import { useEffect, useState, React } from 'react';
+import {
+  Text,
+  FormLabel,
+  Input,
+  FormControl,
+  GridItem,
+  SimpleGrid,
+  Image,
+  VStack,
+  Container,
+  Heading,
+  HStack,
+  InputGroup,
+  InputRightAddon,
+  Button,
+} from '@chakra-ui/react';
+
+const AccountPage = props => {
+  const [isLoading, setLoading] = useState(false);
+
+  // stores information on whether the passwords are hidden/shown, plus the state of the button text
+  const [leftPasswordType, setLeftPasswordType] = useState('password');
+  const [rightPasswordType, setRightPasswordType] = useState('password');
+  const [leftButtonText, setLeftButtonText] = useState('show');
+  const [rightButtonText, setRightButtonText] = useState('show');
+
+  // storing form data in state for retrieval on submission
+  const [changesMade, setChangesMade] = useState(false); // tracks whether user made changes to disable/enable 'Save Changes' button
+  const [firstName, setFirstName] = useState('');
+  const [prefName, setPrefName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [email, setEmail] = useState('');
+  const [trainingStatus, setTrainingStatus] = useState('');
```

Comment:
Training status cannot be changed on the account page, so it should be static and displayed as such